### PR TITLE
feat: lambas sentry transactions

### LINKED ISSUE #3979: Sentry Transactions Iteration #1
https://docs.sentry.io/product/performance/transaction-summary/

**Acceptance Criteria:**
- [x] Wrap Lambdas transactions in scope, the name of the scope should omit query parameters (https://docs.sentry.io/platforms/unity/enriching-events/transaction-name/) as it will create too much irrelevant noise, though they could be included as extra data if needed
- [x] ~~Add Scope for each scene to identify issues per individual scene~~ Done by @m3taphysics 
- [x] ~~Add the number of shown users as extra data per report~~ Done by @m3taphysics 
- [ ] ~~Add a document on how to modify the scope of the report~~ Straightforward

### PR DESCRIPTION
**Pull request by mikhail-dcl**
## What does this PR change?

Closes #3979 

- Added a generic way of tracking `UnityWebRequest` and reporting its result to Sentry
- Applied `WebRequestMonitor` to `LambdasService`, in the future it can be further expanded to any `WebRequest` if have proven useful

...

## How to test the changes?
1. Make Extended Passport Requests (Open Other Users' Profiles)
2. Go to https://sentry.io/organizations/decentraland/performance/
3. You should see reported Requests (keep in mind that the current rate is 5% so only 5% of requests will appear there)
4. Observe "GET nfts/names" and "GET nfts/lands" are reported like this:
image

1. Go to: https://play.decentraland.zone/?renderer-branch={branch_name}&{desired_url_params}
6. ...

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Review by Kinerius — APPROVED** [review]
LGTM

File: unity-renderer/Assets/DCLServices/Lambdas/LambdasService.cs [hunk 1]
```diff
@@ -24,30 +30,35 @@ public class LambdasService : ILambdasService
...
+            return SendRequestAsync<TResponse>(wr, cancellationToken, endPoint, transaction, urlEncodedParams);
         }
 
         public UniTask<(TResponse response, bool success)> Get<TResponse>(
+            string endPointTemplate,
             string endPoint,
-            int timeout = ILambdasService.DEFAULT_TIMEOUT,
-            int attemptsNumber = ILambdasService.DEFAULT_ATTEMPTS_NUMBER,
+            int timeout = 30,
+            int attemptsNumber = 3,
```
Comment (davidejensen) [inline unity-renderer/Assets/DCLServices/Lambdas/LambdasService.cs:43]: Out of curiosity, why this change?

**Review by davidejensen — APPROVED** [review]

File: unity-renderer/Assets/DCLServices/Lambdas/LambdasService.cs (hunk 1 above)
Comment (mikhail-dcl) [reply unity-renderer/Assets/DCLServices/Lambdas/LambdasService.cs:43]: Clean-up changed it for some reason, I will revert

Comment (mikhail-dcl) [reply unity-renderer/Assets/DCLServices/Lambdas/LambdasService.cs]: Ah no, it was automatic refactoring of parameters by Rider, it screwed the things up , that's why I didn't apply it in the Post method
